src/components/ScreenSavers/WindowsXP.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import styled from 'styled-components';
+import logoXP from 'assets/properties/displayProperties/screenSaver/windows-xp-logo.png';
+
+function WindowsXP({
+  winWidth = window.innerWidth,
+  winHeight = window.innerHeight,
+  imgHeight = 175,
+  imgWidth = 240,
+}) {
+  const [logoPosition, setLogoPosition] = useState({ posY: 0, posX: 0 });
+
+  const randomizeLogoPosition = useCallback(() => {
+    let randomPosY = Math.random() * (winHeight - imgHeight);
+    let randomPosX = Math.random() * (winWidth - imgWidth);
+    setLogoPosition({
+      posY: randomPosY,
+      posX: randomPosX,
+    });
+  }, [winWidth, winHeight, imgHeight, imgWidth]);
+
+  useEffect(() => {
+    randomizeLogoPosition();
+    const id = setInterval(() => {
+      randomizeLogoPosition();
+    }, 10000);
```

Comment:
Yay, it's actually 10 seconds!

Reply:
Of course it is not shchona here. It is serious work we doing here..